doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,209 @@
+---
+uid: Overview.Mvux.Advanced.Commands
+---
+
+# Commands
+
+Commands are a way to decouple the UI from the code that performs an action. This makes it easier to maintain and test your code, and it also makes your UI more flexible and adaptable.
+
+A command is an object that implements the [`ICommand`](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.input.icommand) interface. This interface has two methods:
```

Comment:
Two methods **and an event**.  If bothering to explain the methods, why not also explain the event?